### PR TITLE
Added Buglife feature where users can shake phone to showcase any bug…

### PR DESCRIPTION
**Pull request by dignacio0815**
…s in app

### CODE REVIEW

**Comment by dignacio0815** [comment]
i am adding a gif soon just in case if there is not one here yet

File: app/build.gradle [hunk 1]
```diff
@@ -68,7 +69,10 @@ dependencies {
     implementation 'com.google.firebase:firebase-core:17.0.0'
     implementation 'com.firebaseui:firebase-ui-auth:4.3.1'
     implementation 'androidx.recyclerview:recyclerview:1.0.0'
+    implementation 'com.buglife.sdk:buglife-android:1.5.1'
 
+    // Buglife dependencies
+    implementation 'com.buglife.sdk:buglife-android:1.5.1'
```
Comment (chiped) [inline app/build.gradle]: why have it twice?

File: app/src/main/java/com/example/una/BugApplication.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.example.una;
+
+import android.app.Application;
+
+import com.buglife.sdk.Buglife;
+import com.buglife.sdk.InvocationMethod;
+
+public class BugApplication extends Application {
```
Comment (chiped) [inline app/src/main/java/com/example/una/BugApplication.java]: Actually,  this is your application as you specified it in the manifest. Call it `UnaApplication`

File: app/src/main/java/com/example/una/BugApplication.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    @Override
+    public void onCreate() {
+        // Extended Application to use Buglife
+        super.onCreate();
+        Buglife.initWithEmail(this, getString(R.string.denize));
```
Comment (chiped) [inline app/src/main/java/com/example/una/BugApplication.java]: where is this string? Also call it something suitable to the use case, `bug_report_email`?

File: app/build.gradle (hunk 1 above)
Comment (dignacio0815) [reply app/build.gradle]: it does not appear outdated here but i removed the extra dependency of buglife. Sorry about that

**Pull request merged**